### DISCUSSION TITLE
Rnode Bounds - Bandwidth, spreading factor, etc.

### DISCUSSION THREAD

**Discussion by Heavyclouds** [markqvist/Reticulum#553] [Help & Questions] [answered]
I have been playing around with rnodes built off the Lilygo Lora32 boards. In trying to maximize distance for small text packages with Sideband, I ran into some issues. I could not successfully close link at a SF of 10+ with a bandwidth of 62.5 kHz. Once I drop to SF = 9 everything is good. After some digging I found the article [https://www.lab5e.com/docs/lora/dr_sf/](url). This goes into maximum payload size and data rate. There is also the tool on unsigned.io for calculating data rates. 

So my question is, is there a rule of thumb for minimum data rate or payload size to close link and communicate? Thanks!

**Comment by faragher** [comment, marked as answer]
Section 2, specifically 2.5, should provide some good high-level information: [Primer](https://reticulum.betweentheborders.com/primer.pdf)

Rule of thumb: 500 baud is about the floor. You can go much lower in certain circumstances, but that's about the sweet spot.

**Reply by Heavyclouds** [reply]
Thank you! I didn’t know this existed! I was finding the floor through step by step testing and it corresponded perfectly.

**Reply by markqvist** [reply]
To use the very low datarates, boards with temperature-controlled crystal oscillators (TCXOs) are more or less a necessity, since LoRa does have some requirements for frequency stability, and these challenges increase with packet airtime.

As @faragher says, 500 bits per second is about the lowest reliably functioning configuration with most boards, given they're not completely off-kilter in their frequencies. Around 850bps and up seems to work for almost every board out there.

A good balance between speed and range is something like SF8, CR6, BW62.5. That gives you around 1.3 kbps, and quite good range.

**Reply by markqvist** [reply]
Also, an upcoming update to LXMF and Sideband will bring significant improvements to message delivery performance and efficiency for situations like this, where small messages are exchanged directly over low-bitrate mediums.

Since we now have ratchets implemented in Reticulum and LXMF, we can start actually taking advantage of this, for example with efficient, link-less message delivery, that still preserves forward secrecy. Very nice stuff.

**Reply by Heavyclouds** [reply]
Wonderful!
